src/components/Board.vue

```diff
@@ -9,6 +9,7 @@
 		@mousemove="mouseEnterPixel"
 		@mouseleave="dragging = false"
 		@click="clicked"
+		@contextmenu="clearPixel($event)"
```

Comment:
```suggestion
		@contextmenu.prevent="clearPixel"
```
And you wouldn't need the explicit `e.preventDefault()` in the function body

Reply:
You don't need the `($event)` part in any case (even without the `.prevent`)